bundle/Resources/views/admin/tag/search.html.twig

```diff
@@ -0,0 +1,47 @@
+{% extends eztags_admin.pageLayoutTemplate %}
+
+{% trans_default_domain 'eztags_admin' %}
+
+{% block content %}
+    <h1 class="context-title">{{ 'tag.search.search_tags'|trans }}</h1>
+
+    <div class="panel">
+        {{ form_start(form, {'attr': {'novalidate': 'novalidate'}}) }}
+            {{ form_errors(form.searchTerm) }}
+            {{ form_widget(form.searchTerm) }}
+            <button type="submit">{{ 'tag.search.search'|trans }}</button>
+        {{ form_end(form) }}
+
+        {% if tags and tags|length > 0 %}
+            <h2 class="context-title">{{ 'tag.search.search_result_label'|trans({'%searchTerm%': searchTerm, '%numberOfMatches%': tags.nbResults}) }}</h2>
+
+            <table class="list">
+                <tbody>
+                <tr>
+                    <th>{{ 'tag.search.tag_id'|trans }}</th>
```

Comment:
I'm sure there are already existing translations for these strings, used in tag list. No need for duplicating them

Reply:
Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249